content/docs/studio/get-started.md

```diff
@@ -26,11 +31,21 @@ more views._
 4. To create a new view, click on `Add a View`. All the organizations that you
    have access to will be listed.
 
+> If you do not see your desired organizations or Git repositories, make sure
+> that
+> [the connection to your Git server has been set up](/doc/studio/user-guide/account-management#git-integrations).
+
 > To create views from your GitHub repositories, you must install the Iterative
 > Studio GitHub app. Refer to the section on
 > [GitHub app installation](/doc/studio/user-guide/install-github-app) for more
 > details.
 
+> To create views from repositories on your self-hosted GitLab server, you must
```

Comment:
There are special instructions for Bitbucket, and nor for GitLab.com. The special instructions are only for GitHub and for self-hosted GitLab. For this reason (because only a subset of all cases have the special instructions), I'm not creating tabs. I've changed them to `<admon>`.